drivers/serial/Kconfig.bflb

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021-2024 ATL Electronics
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config UART_BFLB
+	bool "Bouffalo Lab serial driver"
+	depends on DT_HAS_BFLB_BL_UART_ENABLED
+	depends on SOC_FAMILY_BFLB
```

Comment:
Depending on `DT_HAS_*_ENABLED` should be enough, so let's remove this depend.

Reply:
Done